### ISSUE TITLE
What's going with the splash-screen?

### ISSUE THREAD

**Issue by a-einstein** [dotnet/maui#12395]
### Description

I am converting from XamarinForms to MAUI, for platforms Android and Windows.  I am trying to get the splash screen working  again on Windows.

Depending of my setup, I get several problems, none of them leading to a desired result.

What's going on?
How to get this working?

### Steps to Reproduce

In the central project-file there is a property like below. I found this described as being the only splash-screen needed. But this only works for Android.
`<MauiSplashScreen Include="Resources\Splash\splash.svg" Color="#58ACFA" BaseSize="350,350" />
`

Then I tried to recreate the setupfor Windows as on XamarinForms.
- Start from scratch in Package.appxmanifest.
- In Visual Assets generate from a png file to Platforms\Windows\Assets and name it 'SplashScreen'. The files get generated, but there are no previews to be seen.
- When starting I get: "error 0x80070003: Cannot install or update package ... because the splash screen image [SplashScreen.png] cannot be located"
- Analogous to XamarinForms I set the build actions to 'Content'. (Why isn't this generated?) Now I get previews.
- After a clean, rebuild and even needed restart of VS, the application runs, but without a splash-screen.
- Analogous to XamarinForms I set the entry point to 'RCS.PortableShop.WinUI.App' instead of '$targetentrypoint$'. (How useful are those placeholders anyway?) Now I get the splash-screen, but the application crashes with "application called an interface that was marshalled for a different thread"

So, not getting anywhere yet.

By the way, even VS crashes sometimes when working on the Visual Assets.

### Link to public reproduction project repository

Commit to be added.

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

windows 10.0.17763.0

### Did you find any workaround?

No.

**Comment by breenbob** [comment]
I've also been struggling with this. Turns out it is not implemented in WinUI yet. Doesn't seem to be high up on the agenda, either. None of the suggested workarounds work like how you'd expect, and are a bit trickier to implement in a Maui WinUI application. Very disappointing. 

See:
https://github.com/dotnet/maui/issues/3870
https://github.com/microsoft/microsoft-ui-xaml/issues/4055

**Comment by Sofiya2696** [comment]
Any update for splash screen to support in UWP?

**Comment by AhsanAli2546** [comment]
any update for splash screen in UWP?

**Comment by Zhanglirong-Winnie** [comment]
Visual Studio Enterprise 17.8.0 Preview 3.0. Can repro on windows platform.

**Comment by NeerajGulia** [comment]
Its been more than an year and still not worked upon.
The another issue without a spalsh screen is that if user click on the app icon on Windows - it can launch more than 1 app in parallel.

**Comment by samhouts** [comment]
We agree that this is an important issue. As our [roadmap ](https://github.com/dotnet/maui/wiki/Roadmap) indicates, for the near future, we are focused on:

- Issues impacting Xamarin.Forms upgrades to .NET MAUI
- CollectionView
- Layout

I am marking this as a p/2 issue because it does not meet our focus areas _right now_, but we are constantly revisiting the backlog for important issues to address as time allows. Thank you for your patience and understanding!

**Comment by nguyenvuhoang** [comment]
Have any solution
Cannot install or update package O24Toolkit_qcvtrzgcjzrs0 because the splash screen image [O24Toolkit.png] cannot be located

**Comment by mattleibow** [comment]
Just a not, WinUI does not have splash screen support: https://github.com/microsoft/microsoft-ui-xaml/issues/4055

You can maybe have a look at some of the comments if you really need to have a splash as there may be some ideas. Otherwise, we have to wait for the WASDK and OS team to come up with something.

**Comment by askariya** [comment]
@breenbob Did you ever happen to find a workaround solution that works for MAUI? I'm struggling with the same thing in 2025.

**Comment by holecekp** [comment]
This error is really annoying. The app worked on Windows but suddenly, without any significant changes, it refuses to start displaying error:
"DEP0700: Registration of the app failed. [0x80073CF6] AppxManifest.xml(54,27): error 0x80070002: Cannot install or update package PACKAGENAME because the splash screen image [splashSplashScreen.png] cannot be located."

I got used to there is no spalshscreen on Windows but I don't understand why it refuses to start when I made no changes related to images or a splashscreen .

**Comment by mihaijoldes** [comment]
> This error is really annoying. The app worked on Windows but suddenly, without any significant changes, it refuses to start displaying error: "DEP0700: Registration of the app failed. [0x80073CF6] AppxManifest.xml(54,27): error 0x80070002: Cannot install or update package PACKAGENAME because the splash screen image [splashSplashScreen.png] cannot be located."
> 
> I got used to there is no spalshscreen on Windows but I don't understand why it refuses to start when I made no changes related to images or a splashscreen .

I fixed this a few times by deleting bin and obj folders, restarting VS(2022) and clean/rebuild. seems something is not cleaned up properly sometimes.

**Comment by bobwhitten** [comment]
> I fixed this a few times by deleting bin and obj folders, restarting VS(2022) and clean/rebuild. seems something is not cleaned up properly sometimes.

Just before I saw this I did "the bin/obj" dance (and also deleting the .vs file, maybe unrelated), and this got better.

**Comment by kubaflo** [comment]
Closing because this is owned by an external platform/tooling dependency rather than actionable MAUI backlog work. If an upstream change reveals a new MAUI-owned bug, please open a fresh issue and link this thread.

**Comment by kubaflo** [comment]
Reopening — this issue should not have been closed.

It carries `s/verified` (a maintainer or partner reproduced it), `s/triaged` (it already went through triage), `partner/winui` (a partner team is tracking it). Those labels mean the report was already confirmed as a real product defect, so it is not eligible for backlog cleanup closure. Closure here requires specific fix evidence (a merged PR that closes it, or a `fixed-in-*` label), and there was none.

Sorry for the noise, and thanks to everyone who flagged it.

> Disclosure: this comment was generated by AI.
